libs/sdk-bindings/makefile

```diff
@@ -30,19 +30,30 @@ darwin-universal: $(SOURCES)
 	lipo -create -output ../target/darwin-universal/release/libbreez_sdk_bindings.dylib ../target/aarch64-apple-darwin/release/libbreez_sdk_bindings.dylib ../target/x86_64-apple-darwin/release/libbreez_sdk_bindings.dylib
 	lipo -create -output ../target/darwin-universal/release/libbreez_sdk_bindings.a ../target/aarch64-apple-darwin/release/libbreez_sdk_bindings.a ../target/x86_64-apple-darwin/release/libbreez_sdk_bindings.a
 
+csharp-darwin: darwin-universal
+	cargo install uniffi-bindgen-cs --git https://github.com/NordSecurity/uniffi-bindgen-cs
+	uniffi-bindgen-cs src/breez_sdk.udl -o ffi/csharp -c ./uniffi.toml 
+	cp ../target/darwin-universal/release/libbreez_sdk_bindings.dylib ffi/csharp
+
+TARGET ?= aarch64-unknown-linux-gnu
+csharp-linux: $(SOURCES)
```

Comment:
I tested `make csharp-linux` and it worked.

I did have to change the default target to `x86_64-unknown-linux-gnu`, but each tester can override it with `--target` based on his own local architecture and available rust targets.